packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -15,6 +15,14 @@ import 'text.dart';
 import 'ticker_provider.dart';
 import 'transitions.dart';
 
+// Examples can assume:
+// class MyWidget extends ImplicitlyAnimatedWidget {
+//   MyWidget() : super(duration: const Duration(seconds: 1));
+//   final Color targetColor = Colors.black;
```

Comment:
Can you add a blank line between members for clarity?

Reply:
Unfortunately, that seems not to be supported by the sample tool. When I had a blank line here, the analyzer step failed with `Error: Unexpected content in sample code preamble.`